merkle/smt/tiles.go

```diff
@@ -97,6 +98,9 @@ func (t *TileSetMutation) Build() ([]Tile, error) {
 	for id, upd := range t.tiles {
 		had, ok := t.read.tiles[id]
 		if !ok {
+			sort.Slice(upd, func(i, j int) bool {
```

Comment:
This `Sort` can go away if we specify the order in which nodes can be `Set` (it is implicitly assumed that they are sorted now), or only allow inserting a whole tile at once.

Will add a TODO.

Reply:
Done.